packages/cactus-cmd-gui-app/README.md

```diff
@@ -0,0 +1,48 @@
+# `@hyperledger/cactus-cmd-gui-app`
+
+This component allows viewing ledger data in Supabase. The data is fed to supabase by persistence plugins for each ledgers.
+
+## Summary
+
+- [Remarks](#remarks)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
+- [License](#license)
+- [Acknowledgments](#acknowledgments)
+
+## Remarks
+
+- Plugin requires running Supabase and persistence plugins in order to properly view ledger data.
```

Comment:
@ervvkb Would it be possible to include shell commands here on how to get that done as well? Ideally the steps that can be executed in order and then have the whole thing working.

I see that there is a "Usage" section already but it keeps it vague by saying "use this and that image" and "set this paramtere to the "correct" value" (I'm guessing it's self-evident what is the correct value if you have experience like you do, but to others that may be less helpful to just say "set it to the correct value")

It is also fine if you do it later, but then I'll ask that you create a specific issue in the tracker for it with a title something like `docs(cmd-gui-app): add usage guide to README`